docs/getting-started-guides/mesos/index.md

```diff
@@ -324,13 +324,13 @@ Future work will add instructions to this guide to enable support for Kubernetes
 
 [1]: https://docs.mesosphere.com/latest/usage/service-guides/hdfs/
 [2]: https://docs.mesosphere.com/latest/usage/service-guides/spark/
-[3]: https://docs.mesosphere.com/latest/usage/service-guides/chronos/
-[4]: https://releases.k8s.io/{{page.githubbranch}}/cluster/addons/dns/README.md
+[3]: https://mesos.github.io/chronos/docs/getting-started.html
+[4]: http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/dns/README.md
```

Comment:
{{page.githubbranch}} doesn't automatically transform here.
Change https to http doesn't work.
Should I import something to this page?